java/vector/src/main/codegen/templates/HolderReaderImpl.java

```diff
@@ -131,6 +127,15 @@ public void read(Nullable${name}Holder h) {
     byte[] value = new byte [holder.byteWidth];
     holder.buffer.getBytes(0, value, 0, holder.byteWidth);
     return value;
+  <#elseif minor.class == "TimeStampSec">
```

Comment:
These are also changing return value of API. Applications will break that depend on these reader APIs. You may want to update release notes to reflect this change in readers.